### PR TITLE
Fix the Travis build with multiple recipes

### PR DESCRIPTION
**Pull request by stof**
| Q             | A
| ------------- | ---
| License       | MIT

#SymfonyConHackday2018

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,4 +23,4 @@ install:
     - cd flex
 
 script:
-    - composer req "${PACKAGES}"
+    - composer req $(echo $PACKAGES)
```
[the remark this answers is not in the document]
Comment (Nyholm) [reply .travis.yml]: No, then all the line breaks in $PACAKGES will be printed.

[the remark this answers is not in the document]
Comment (Nyholm) [reply .travis.yml]: (I asked the same question)

[the remark this answers is not in the document]
Comment (stof) [reply .travis.yml]: The previous code was passing all packages as a single argument to Composer. I tried without the double quotes, but the same happens (it looks like bash has some magical escaping there, for some reasons). That's the only way I found to have it working.

[the remark this answers is not in the document]
Comment (B-Galati) [reply .travis.yml]: Makes sense, thanks! :+1:

**Pull request merged**